tomviz/ModuleVolume.cxx

```diff
@@ -154,7 +154,7 @@ bool ModuleVolume::deserialize(const pugi::xml_node& ns)
     }
   }
 
-  return this->Superclass::deserialize(ns);
+  return Module::deserialize(ns);
```

Comment:
Curious: why do you prefer `Module` here instead of `Superclass` (which is typdefed to `Module`)?